GameServerLibTests/Tests/Items/TestInventoryManager.cs

```diff
@@ -11,8 +13,9 @@ public class InventoryManagerTests
         [TestMethod]
         public void TestAddAndGetItem()
         {
-            var itemManager = new ItemManager();
-            itemManager.AddItems(ItemContentCollection.LoadItemsFrom("../../../Content"));
+            var game = new Game();
+            var itemManager = game.ItemManager;
```

Comment:
Why to create `Game` to check the `ItemManager`? Change it to just creation of `ItemManager`.